todo_app/app.py

```diff
@@ -1,24 +1,29 @@
 from flask import Flask, render_template, request, redirect, url_for
 from todo_app.data.trello_items import get_items, add_item, complete_item
+from todo_app.view_models.todo_list_view_model import TodoListViewModel
 
 from todo_app.flask_config import Config
 
-app = Flask(__name__)
-app.config.from_object(Config())
+def create_app():
+    app = Flask(__name__)
+    app.config.from_object(Config())
 
+    @app.route('/')
+    def index():
+        todoItems = get_items()
+        app.logger.info(todoItems)
```

Comment:
This looks like pretty useless output in my terminal - it just looks like an array with a series of `<todo_app.data.item.Item object at address>`!

We can make that a bit nicer with the magic `__repr__` function on a class if we want to, for example inside your `Item` class if we add:
```python
    def __repr__(self):
        return f'Ticket {self.id} is in status {self.status}'
```
Then we'll get much more human readable logging here. You'll notice I've avoided adding any reference to the Item name - very intentionally! That could easily end up with some personal data (e.g. "take <person> to see GP about <medical condition>" or "send email to the king at <private email address>" or something!) and we generally want to keep that out of our logging, so we can confidently share logs with people that we wouldn't necessarily give production database access to for example

Reply:
ahh I accidentally left this in when debugging but this is really helpful information! Thanks :))